### ISSUE TITLE
Lifecycle Tasks

### ISSUE THREAD

**Issue by pi0** [nitrojs/nitro#2021]
On top of Tasks API (#1974), we can support specific task prefixes `init:` (blocker), `startup:` (non-blocker), and `close:` that will be automatically called by nitro during startup and server close.

**Comment by go4cas** [comment]
@pi0 ... will this include scheduled (cron) tasks as well?

**Comment by pi0** [comment]
Yes but tracker for that is https://github.com/unjs/nitro/issues/1974

**Comment by notKamui** [comment]
What is the current state of this issue ?
